python/lsst/sims/photUtils/Sed.py

```diff
@@ -67,8 +67,8 @@
   resampleSED -- primarily internal use, but may be useful to user. Resamples SED onto specified grid.
   flambdaTofnu / fnuToflambda -- conversion methods, does not affect wavelen gridding.
   redshiftSED -- redshifts the SED, optionally adding dimmingx
-  setupCCMab / addCCMDust -- separated into two components, so that a_x/b_x can be reused between SEDS
-if the wavelength range and grid is the same for each SED (calculate a_x/b_x with setupCCMab).
+  setupODonnell_ab / setupCCMab / addDust -- separated into two components, so that a_x/b_x can be reused between SEDS
```

Comment:
Regarding the name:

I don't want to change `setupCCMab` because of backwards compatibility, but I don't like the look of `setupODonnellab`.  Do you mind if I leave the inhomogeneous use of `_`?

Reply:
Alternatively, I can add a `setupCCM_ab` and have `setupCCMab` just call `setupCCM_ab`.

This code is so old, I am leary of breaking backwards compatibility in any way.